modules/core/core-api/src/main/java/com/enonic/xp/image/ImageHelper.java

```diff
@@ -85,6 +85,44 @@ public static ImageWriter getWriterByFormat( final String format )
         }
     }
 
+    public static String getFormatByMimeType( final String mimeType )
```

Comment:
Format is only part of hash, right? If so, it should be private

Reply:
no, we have deprecated api method to fetech it.